### DISCUSSION TITLE
Support `src` directory

### DISCUSSION THREAD

**Discussion by ceopaludetto** [onejs/one#356] [Ideas]
Add support to place application files in `src/` directory. Remix allow users to pass a appDirectory in vite plugin configuration. I would be awesome to also rename app/ to something else. I do like `src/routes` instead of just plain `app/`

**Comment by arthur-fontaine** [comment]
+1

The `src` directory is almost a must in all my projects.

[Expo Router supports it too.](https://docs.expo.dev/router/reference/src-directory/)

Maybe a config option that specifies the entry directory in the Vite plugin could be added?
